doc/nrf/gs_ins_windows.rst

```diff
@@ -83,20 +83,22 @@ The |NCS| consists of the following repositories:
 Every |NCS| release consists of a combination of these repositories at different revisions.
 
 .. note::
-   The latest state of development is on the master branches of the repositories.
-   Note that this might not be a stable state.
+   The latest state of development is on the master branch of the fw-nrfconnect-nrf repository.
    To ensure a usable state, the fw-nrfconnect-nrf repository defines the compatible states of the other repositories.
-   However, this state is not necessarily tested.
+   However, this state is not necessarily tested. Releases and tags however do guarantee a higher quality level.
 
    Therefore, unless you are familiar with the development process, you should always work with a specific release of the |NCS|.
 
 To manage the combination of repositories and versions, the |NCS| uses :ref:`zephyr:west`.
-The main repository, fw-nrfconnect-nrf, contains a `west manifest file`_ that defines the versions of all repositories.
-This means that fw-nrfconnect-nrf acts as manifest repository (see :ref:`Repository structure <zephyr:west-struct>`), while the other repositories are project repositories.
-The version of the manifest file defines what versions of the project repositories are checked out.
-
-In this way, you can decide to work with a specific |NCS| release either by initializing west with the manifest file for the release or by checking out the manifest file for the release and updating your project repositories to the corresponding state.
-Alternatively, you can work with the latest state of development by using the manifest file from the master branch of the fw-nrfconnect-nrf repository.
+The main repository, fw-nrfconnect-nrf, contains a `west manifest file`_ that defines the versions of all other repositories.
+This means that fw-nrfconnect-nrf acts as the manifest repository (see :ref:`Repository structure <zephyr:west-struct>`), while the other repositories are project repositories.
+The revision (in Git terms) of the fw-nrfconnect-nrf repository determines the contents of the manifest file. This file in turn defines what versions of the project repositories are checked out.
```

Comment:
This is only true, as long as there are no local modifications to the manifest file.
Maybe this should be mentioned somewhere, although I don't think this is the right place.

This is actually different from `git submodule update` where you'll always get what is commited in the repo.
With `west.yml` user can make local mods and get another version of project.

Reply:
New line for second sentence.

Reply:
I really do not want to introduce the concept of modifying the `-nrf` manifest file by hand. This is a Getting Started guide. I am sure that users willing to play with west will figure that out by looking at the west documentation in Zephyr, but in this section I want to keep things simple

Reply:
I was more thinking in-line of some kind of warning, that if they do have local modification, then they will not get the revision specified in the repo.

Reply:
Sure but what I mean is that this is already documented in the west docs that we link to, and this page is really just a very high-level introduction to the concepts of manifest repository and project repositories.